code/modules/projectiles/guns/ballistic/shotgun.dm

```diff
@@ -2,6 +2,8 @@
 	name = "shotgun"
 	desc = "A traditional shotgun with wood furniture and a four-shell capacity underneath."
 	icon_state = "shotgun"
+	worn_icon_state = null
+	worn_icon = 'icons/mob/clothing/back.dmi'
```

Comment:
handled by build_worn_icon

Reply:
Reverted. Had the worn_icon set to make the other slots use a working icon. I'll add it to the list of missing textures.